### DISCUSSION TITLE
How can we enable light probes and reflection probes for the character?

### DISCUSSION THREAD

**Discussion by rebeccakzpt** [naverz/zepeto-studio-kor#739] [5. Q&A]
The lighting in my world is complicated. It's impossible to use real-time lighting for all light sources. But the character doesn't have light probes and reflection probes turned on by default. It looks very dark. Is there a way to turn them on?

**Comment by Zepeto-TechEvangelist** [comment]
Hello, You have to turn on all the blend light probe options of the skinned MeshRender.
Use the code below to find and turn on all Skinned Mesh Render in the character.
please create a new ZEPETO script and copy and paste the contents below

<hr/>

import { SkinnedMeshRenderer } from 'UnityEngine';
import { LightProbeUsage } from 'UnityEngine.Rendering';
import { ZepetoPlayers } from 'ZEPETO.Character.Controller'
import { ZepetoScriptBehaviour } from 'ZEPETO.Script'

export default class LightProbeEnabler extends ZepetoScriptBehaviour {

    Start() {
        ZepetoPlayers.instance.OnAddedPlayer.AddListener((userID) => {
            let characterContext = ZepetoPlayers.instance.GetPlayerWithUserId(userID).character.Context;
            let skinnedMeshRenderers = characterContext.GetComponentsInChildren<SkinnedMeshRenderer>();
            skinnedMeshRenderers.forEach(smr => {
                smr.lightProbeUsage = LightProbeUsage.BlendProbes;
            });
        })
    }
}

![스크린샷 2022-05-12 오후 4 40 13](https://user-images.githubusercontent.com/100118650/168017921-738bf90a-ba89-4a4d-9337-499ef52656bc.png)

**Reply by rebeccakzpt** [reply]
THANK YOU SOOOOOOOOO MUCH!!!
